.circleci/docker/Dockerfile.ubuntu1904

```diff
@@ -41,14 +41,7 @@ RUN set -ex; \
 	pip install codecov; \
 	rm -rf /var/lib/apt/lists/*
 
-# Aleth for end-to-end tests
-ARG ALETH_VERSION="1.6.0"
-ARG ALETH_HASH="7f7004e1563299bc57882e32b32e4a195747dfb6"
-ARG ALETH_URL="https://github.com/ethereum/aleth/releases/download/v${ALETH_VERSION}/aleth-${ALETH_VERSION}-linux-x86_64.tar.gz"
-RUN set -ex; \
-	wget -q -O /tmp/aleth.tar.gz "${ALETH_URL}"; \
-	test "$(shasum /tmp/aleth.tar.gz)" = "$ALETH_HASH  /tmp/aleth.tar.gz"; \
-	tar -xf /tmp/aleth.tar.gz -C /usr
```

Comment:
This PR is about optimizing the docker image by utilizing stages. Why are we removing the Aleth binary here? (is EVMONE already enabled by default?) @chriseth :)

Reply:
I added aleth back for now.